### ISSUE TITLE
shortcut keys bug

### ISSUE THREAD

**Issue by HiSDR** [igrigorik/videospeed#871]
It works well, but configuring shortcut keys seem a little buggy。

**Comment by thewheat** [comment]
Hi @HiSDR , thanks for the report but seems that I can't replicate the problem.

Could we get a bit more context on this and perhaps any steps to reproduce the problem? I've been able to change shortcuts and they work on my end. I've documented my steps below. Is it a specific shortcut that isn't working?

**Steps**
- Open Options
- Click into shortcut to change and press new shortcut
- Click save
- Load page with a video (e.g. https://www.bbc.com/news/world-europe-58610234 ) and new shortcut should register
![image](https://user-images.githubusercontent.com/892961/133922771-78116048-722f-42a0-8b72-8cab325bded7.png)

I'm using 
- Video Speed Controller Extension version `0.6.5`
- Chrome `93.0.4577.82 (Official Build) (x86_64)`
-  MacOS `11.6 (20G165)`

**Comment by HiSDR** [comment]
I'm using
    Video Speed Controller Extension version 0.6.5
    Chrome 93.0.4577.82 (Official Build) (x86_64)
    MacOS 11.6 (20G165)
[image: 截屏2021-09-19 18.05.39.png]
1-3 are my steps. 4 is the prompt after saving.
1-3 are my steps.
4 is the prompt after saving
Please note that at this time, the shortcut keys for refreshing the page
will restore to the default letters, or you can watch the video without
refreshing the shortcut keys to modify the default letters or not.
Only the numbers modified in step 2 can be stored.

This is true in Microsoft Edge, Vivaldi and other browsers that use Chrome
extensions.
[image: 截屏2021-09-19 18.30.18.png]
In addition, the author of Video Speed Controller on Firefox is
codebicycle. The same is true. I personally think that these two extensions
are the same author or one of them plagiarizing the other.


Timothy Lim ***@***.***> 于2021年9月19日周日 下午5:48写道：

**Comment by HiSDR** [comment]
Thanks for your reply, I think I can use it correctly.

- I have sorted out the problem-------the special input method I used caused this unpleasant time.
- RIME, it will quickly input English characters (Shift+any word) in the Chinese input state, which is not recognized by this extension, causing the shortcut keys to fail to save but the numbers are OK. I have reported this issue to the developer.
- During the use, I found that other people in the chrome app store also reported the same problem. I pasted the link as a reference. Hope to help latecomers.

- [https://github.com/rime/home/issues/885]
